CIP-0088/CIP-25/README.md

```diff
@@ -0,0 +1,257 @@
+
+This entry should follow the "URI Array" syntax where the URI scheme is the first string entry of the array followed by
+one or more strings comprising the path to the asset. In this way we can support HTTP, HTTPS, Arweave, and IPFS in a 
+consistent fashion for maximum flexibility. Consumer projects should, where possible, cache these images locally to 
+ensure image availability and consistent consumer experience until an updated registration is observed.
+
+This image is generally used similar to a "PFP" type of image so should be in a square aspect ratio (1:1) and padded to
+accommodate cropping depending on the platform.
+
+**Example:**
+
+```json
+{
+  "25|68": {
+    "1": {
+      "2": [
+        "https://",
+        "static.spacebudz.io",
+        "/images/logo.png"
+      ]
+    }
+  }
+}
+```
+
+### Index 3 (Project Banner Image)
+
+##### Type: URI Array | Required: No | Default: None
+
+Following the same format for the Project Image URI Array above, this URI should point to a "banner image" that can be
+used to represent the project and optionally be displayed by platforms.
+
+The specified resource for both Project Image and Project Banner Image should be an image that is natively supported by
+the HTML `<img>` tag.
+
+**Example:**
+
+```json
+{
+  "25|68": {
+    "1": {
+      "3": [
+        "https://",
+        "static.spacebudz.io",
+        "/images/banner.jpg"
+      ]
+    }
+  }
+}
+```
+
+### Index 4 (Not Safe for Work [NSFW] Flag)
+
+##### Type: Unsigned Integer | Required: No | Default: 0
+
+This is a simple true/false field (1 = True, 0 = False, Default = 0) to represent whether the project contains
+"Not Safe for Work" content that may not be suitable for all audiences.
+
+**Example:**
+
+```json
+{
+  "25|68": {
+    "1": {
+      "4": 0
+    }
+  }
+}
+```
+
+### Index 5 (Social Media URIs)
+
+##### Type: Object | Required: No | Default: None
+
+This will be the only field where we will use named keys because social media platforms are so varied and rapidly
+evolving. The object should consist of key:value pairs where the name of the platform should be the key and a URI Array
+pointing to the project's social media profile is the value. Keys should always be all lowercase and consist of only
+letters and numbers.
+
+**Example:**
+
+```json
+{
+  "25|68": {
+    "1": {
+      "5": {
+        "twitter": [
+          "https://",
+          "twitter.com/spacebudzNFT"
+        ],
+        "discord": [
+          "https://",
+          "discord.gg/spacebudz"
+        ]
+      }
+    }
+  }
+}
+```
+
+### Index 6 (Project Name)
+
+##### Type: String | Required: No | Default: Policy Name
+
+For projects that belong under an over-arching umbrella (such as Claymates, Virtua, etc.) where a single "project" may
+have multiple NFT policies or "collections", this field allows a space for the "Project Name". This could also be the
+name of the "artist" if an artist creates NFTs under multiple different policies.
```

Comment:
I don't know if the term "mint their art" is term most commonly used but I agree with this change in theory.

Reply:
I have attempted everywhere that CIP-25 or CIP-68 are mentioned to change the language to reflect a "token project" rather than an "NFT" project specifically.